docs/openfaas-pro/dashboard.md

```diff
@@ -111,6 +111,22 @@ spec:
 
 You can use the same tunnel and exit server for multiple domains, to expose both the gateway and the dashboard with TLS and authentication.
 
+If you only plan on accessing the dashboard using port-forwarding, you can set the public url in your values.yaml file to `localhost` or an empty string.
```

Comment:
> If you only plan on accessing the dashboard using port-forwarding

I would change this to: 

"If you don't want to expose your dashboard to users, then you can access it as and when required using port-forwarding. Instead of giving a domain and DNS record, you can..."

So that we can explain the use-case along with the method.